metrics/send_metrics.go

```diff
@@ -0,0 +1,23 @@
+package metrics
+
+import (
+	"flag"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"log"
+	"net/http"
+)
+
+
+func Init() {
+	go send()
+}
+
+func send() {
+	addr := flag.String("listen-address", ":9999", "The address to listen on for HTTP requests.")
```

Comment:
let's read this from the existing config file and instead of reading string, let's read port number as integer and build the expected string format in the code.